option optimize_for = LITE_RUNTIME;

message CrusadeFortressBuyBuffAnswer
{
 required uint32 result = 1;
 required uint32 id = 2;
 required uint32 buy_cnt = 3;
}
